src/components/HeaderForm.js

```diff
@@ -68,6 +76,40 @@ const EMPTY_STATE = {
   bedSelectOptions: ["none"],
 };
 
+// Creates track to be stored in ViewTarget
+// Modify as the track system changes
+// INPUT: file structure, see Types.ts
+function create_track(file) {
+  //track properties
+  const files = [file]
+
+  //remove empty files here?
+
+  const track = {
+    files: files,
+  };
+  return track;
+}
+
+// Checks if all file names in the track are equal
+function tracks_equal(curr, next) {
```

Comment:
Do we not care if the types of the tracks differ?

Could we maybe use `isEqual` from `react-fast-compare` which we already depend on? Or do we need our own function so we can later ignore settings changes on the tracks that won't affect the downloaded data?